docs/topics/cli-args.adoc

```diff
@@ -38,7 +38,15 @@ This option may result in a longer execution time and a large number of migratio
 |--enableClassNotFoundAnalysis |Flag to enable analysis of Java files that are not available on the class path. This should not be used if some classes will be unavailable at analysis time.
 |--enableCompatibleFilesReport |Flag to enable generation of the Compatible Files report. Due to processing all files without found issues, this report may take a long time for large applications.
 |--enableTattletale |Flag to enable generation of a Tattletale report for each application. This option is enabled by default when `eap` is in the included target. If both `enableTattletale` and `disableTattletale` are set to true, then `disableTattletale` will be ignored and the Tattletale report will still be generated.
-|--excludePackages |A space-delimited list of packages to exclude from evaluation. For example, entering `com.mycompany.commonutilities` would exclude all classes whose package name begins with `com.mycompany.commonutilities`.
+|--enableTransactionAnalysis |[Technology Preview] Flag to enable generation of a Transactions Report that displays the call stack, which executes operations on relational database tables. It supports Spring Data JPA and the traditional preparedStatement() method for SQL statement execution. It does not support ORM frameworks, such as Hibernate.
+
+[NOTE]
+====
+enableTransactionAnalysis is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+====
+
+|--excludePackages |A spaceenableTransactionAnalysis-delimited list of packages to exclude from evaluation. For example, entering `com.mycompany.commonutilities` would exclude all classes whose package name begins with `com.mycompany.commonutilities`.
```

Comment:
1. I think you mean "a space-delimited list"...
2. Consider replacing "would exclude" with "excludes." We want to use present as much as possible -- it's always good for facts, even ones in the future, such as "Tomorrow's sunrise is at 5:23."